test/parallel/test-util-callbackify.js

```diff
@@ -0,0 +1,49 @@
+'use strict';
+const common = require('../common');
+
+
+const assert = require('assert');
+const { callbackify } = require('util');
+
+
+{
+  const sentinel = 'hello world';
```

Comment:
https://en.wikipedia.org/wiki/Sentinel_value adopted in TDD (at least back in the day) as an Object you use to guard with...